### ISSUE TITLE
Accessibility (A11y): Resolve violations reported by the "IBM Equal Access Accessbility Checker": Link in canvas

### ISSUE THREAD

**Issue by tomlyn** [elyra-ai/canvas#2461]
Ref: 4031
